### PR TITLE
Beta

### PR DESCRIPTION
**Pull request by SOF3**

### CODE REVIEW

**Comment by Awzaw** [comment]
All good for me now :-)

**Comment by SOF3** [comment]
Actually what is this branch for?

**Comment by Awzaw** [comment]
It was my fork of master to add what I thought sI could do for beta....

**Comment by archie426** [comment]
@SOF3 If you mean the 'PR: Branch' label, it is what is says on the tin, really

File: src/poggit/model/PluginRelease.php [hunk 1]
```diff
@@ -124,4 +132,197 @@ class PluginRelease {
...
+        $result = array_search($api, array_keys(PocketMineApiInfo::$VERSIONS));
+        if($result === false) throw new SubmitException("Unknown API version $api");
+        return $result;
+    }
+
+    public function submit(): int {
+        $buildId = Poggit::queryAndFetch("INSERT INTO releases 
+            (name, shortDesc, artifact, projectId, buildId, version, description, changelog, license, licenseRes, flags, creation, state) 
+            VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)", str_replace(" ", "",
+            " s        s         i          i         i        s          i           i         s          i        i       i        i  "),
```
Comment (Magicode1) [inline src/poggit/model/PluginRelease.php]: Lol

**Comment by SOF3** [comment]
Warning: created new database changes

**Pull request merged**